source/includes/examples/_find-the-center.html.md

```diff
@@ -0,0 +1,182 @@
+
+The `PlayerMove` schema has one record, `delta`, assigned the constrained type of Int32 to allow for negative and positive values.
+
+```inkling
+schema SimConfig
+    Int32 dummy
+end
+```
+
+The `SimConfig` schema has one record, `dummy`, because there is no configuration needed in this particular example.
+
+###### Concept
+
+```inkling
+concept find_the_center
+    is classifier
+    predicts (PlayerMove)
+    follows input(GameState)
+    feeds output
+end
+```
+
+This concept is named `find_the_center`, and it takes input from the simulator about the state of the game (`GameState` schema). It outputs to the `PlayerMove` schema. This is the AI's next move in the simulation.
+
+###### Simulator
+
+```inkling
+simulator find_the_center_sim(SimConfig)
+    action (PlayerMove)
+    state (GameState)
+end
+```
+
+The `simulator` is called `find_the_center_sim` (shown in #simulator-file) and takes the schema input of `SimConfig` (even though it isn't configuring anything, it's required by the simulator). The simulator is going to output the `action` and the `state` of the simulator which is sent to the lesson.
+
+###### Curriculum
+
+```inkling
+curriculum find_the_center_curriculum
+    train find_the_center
+    with simulator find_the_center_sim
+    objective time_at_goal
+        lesson seek_center
+            configure
+                constrain dummy with Int32{-1}
+            until
+                maximize time_at_goal
+end
+```
+
+The curriculum is named `find_the_center_curriculum`, and it trains the `find_the_center` concept using the `find_the_center_sim`. This curriculum contains one lesson, called `seek_center`. It configures the simulation, by setting a number of constraints for the state of the simulator.
+
+The lesson trains until the AI has maximized the objective `time_at_goal`.
+
+
+## Simulator File
+
+```python
+import bonsai
+import sys
+from bonsai.simulator import SimState
+from random import randint
+
+""" If you would like to debug your code add this back in.
+def debug(*args):
+    print(*args, file=sys.stderr)
+"""
+
+
+class BasicSimulator(bonsai.Simulator):
+    """ A basic simulator class that takes in a move from the inkling file,
+    and returns the state as a result of that move.
+    """
+
+    min = 0
+    max = 2
+    goal = 1
+
+    def __init__(self):
+        super(self.__class__, self).__init__()
+        self.goal_count = 0
+        self.value = randint(self.min, self.max)
+        self.old_value = self.min
+
+    def get_terminal(self):
+        """ Function to restart the simulation if the Inkling move was out of bounds.
+        """
+        if (self.value < self.min or self.value > self.max):
+            # (self.value == self.goal and self.old_value == self.goal)):
+
+            # debug("terminal")
+            self.reset()
+            return True
+        else:
+            return False
+
+    def start(self):
+        """ Function to start the episode by guessing a random integer between
```

Comment:
I would not use the word "guessing"... as the AI is not making a move. Instead consider:

""" Start the episode by initializing value to a random number between the min and max. """